app.js

```diff
@@ -18,7 +20,10 @@ let key;
 const keyPath = `${appPath}/key.txt`;
 try { key = fs.readFileSync(keyPath) } catch (_) {}
 
-const archive = drive.createArchive(key, { live: true });
+const archive = drive.createArchive(key, {
+  live: true,
+  file: name => raf(`${appPath}/files/${name}`)
```

Comment:
I'm not sure putting it in the downloads folder is the right idea. When you have limited disk like I do you frequently purge it. On the other hand, that's exactly the right thing if you just use this for experimentation for now - then the data you fetch is short lived anyways.

> this points to the electron folder right?

Yup, that's in `Application Support`.

Hm still `~/Downloads` is probably the best folder. A quick npm search didn't reveal any module for it.

What do others think?